binary_transparency/firmware/devices/usb_armory/README.md

```diff
@@ -36,3 +36,89 @@ directory:
 ```bash
 make CROSS_COMPILE=arm-none-eabi- TARGET=usbarmory imx BOOT=uSD START=5242880
 ```
+
+
+Firmware images
+---------------
+
+Currently, the bootloader can only chain to a Linux kernel.
+
+There are some invariants which must hold for this chain to work:
+1. The `firmware` partition MUST be located at the precise offset mentioned
+    above.
+2. The `firmware` partition MUST be formatted with ext4.
+3. The `firmware` partition MUST contain a `/boot` directory with at least the
+    following contents:
+    * `armory-boot.conf` - a JSON file which tells the bootloader which files
+      to load.
+    * a valid ARM linux Kernel image
```

Comment:
nit this bullet doesn't end with a period and the others do. Recommend removing period on the others, but you can add one to this if that's your jam.

Reply:
ta, done